apps/s_client.c

```diff
@@ -935,6 +935,7 @@ int s_client_main(int argc, char **argv)
     int srp_lateuser = 0;
     SRP_ARG srp_arg = { NULL, NULL, 0, 0, 0, 1024 };
 #endif
+    char *srtp_profiles = NULL;
```

Comment:
Why not put this inside an #ifdef too?

Reply:
Because it becomes just a local variable now instead of a global one.
The two places you note must be guarded together (or not),  else compilers will raise an unused variable warnings.
Disabling the option declaration and the use place of `srtp_profiles` is enough, if we can afford an unused pointer.

Reply:
I would prefer to get an error that it's not supported instead of parsing something and then just ignoring it.

Reply:
@kroeckx : I just try to simplify the handling of NO_SRTP and make it more similar to the other options like NO_SRP, or NO_PSK ...
None of the disable options are actually triggering an error message.